client/my-sites/media-library/list-item.tsx

```diff
@@ -1,53 +1,59 @@
-/** @format */
-
 /**
  * External dependencies
  */
-
-import { assign, isEqual, noop, omit } from 'lodash';
-import PropTypes from 'prop-types';
+import { isEqual, noop } from 'lodash';
 import React from 'react';
 import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import Spinner from 'components/spinner';
-import Gridicon from 'gridicons';
+import Gridicon from 'components/gridicon';
```

Comment:
Should "raw" `gridicons` import be prohibited?

Reply:
I didn't want to change the whole codebase when I added `components/gridicon`, so I took care of it with a webpack alias, so in practical terms it's the same.

I do think that we shouldn't be using the "raw" import in new code, though, as it's misleading. Do you think we should add a custom lint rule?

Reply:
A lint rule might be good, we've already got one for redux's `combineReducers`. I was going to suggest piggy-backing on the implementation, but we might just switch both of them to use [`no-restricted-imports`](https://eslint.org/docs/rules/no-restricted-imports)

https://github.com/Automattic/wp-calypso/blob/25ec4f62a1ee816f8abf9957b2a55f293309ad05/.eslintrc.js#L126
https://github.com/Automattic/wp-calypso/blob/e5ff4bb5fd98726827b35ae261908bc0afd43b6e/packages/eslint-plugin-wpcalypso/lib/rules/import-no-redux-combine-reducers.js#L24-L33

Reply:
Sounds good! 👍 I'll prepare a PR.